build.gradle

```diff
@@ -90,10 +90,9 @@ dependencies {
     compile 'com.jgoodies:jgoodies-common:1.8.1'
     compile 'com.jgoodies:jgoodies-forms:1.9.0'
 
-    // update to 2.0.x is not possible - see https://github.com/JabRef/jabref/pull/1096#issuecomment-208857517
-    compile 'org.apache.pdfbox:pdfbox:1.8.13'
-    compile 'org.apache.pdfbox:fontbox:1.8.13'
-    compile 'org.apache.pdfbox:jempbox:1.8.13'
+    compile 'org.apache.pdfbox:pdfbox:2.0.8'
+    compile 'org.apache.pdfbox:fontbox:2.0.8'
```

Comment:
Further below (starting at line 218), we added exceptions for the update dependencies task. These are now invalid and should be removed.

Reply:
DONE